crates/network/src/lib.rs

```diff
@@ -51,6 +54,8 @@ pub struct BlockResponse<N: Network> {
 }
 
 /// Captures type info for network-specific RPC requests/responses.
+// todo: block responses are ethereum only, so we need to include this in here too, or make `Block`
+// generic over tx/header type
```

Comment:
let's do as follow-up IMO when we're adding the 2nd network?

Reply:
https://github.com/alloy-rs/alloy/issues/267